### ISSUE TITLE
tnnl.ink is down

### ISSUE THREAD

**Issue by alexellis** [LiljebergXYZ/tnnlink#1]
Hi there, nice side project. 

tnnl.ink is down? Have you had a look at https://inlets.dev yet?

Alex

**Comment by faejr** [comment]
Hello! 

Yes that's right, I didn't find the time to maintain the project any longer so I decided to let tnnl.ink go for now. 

I had not seen inlets before, it definitely looks like a nice alternative :)

**Comment by RahulSDeshpande** [comment]
@alexellis
But there is no free forever plan on inlets.dev site.
Are you planning to bring that up any time soon?

**Comment by alexellis** [comment]
No, but you can try it for free before buying your subscription. 🙂
